src/coreComponents/constitutive/fluid/PVTFunctions/EzrokhiBrineDensity.cpp

```diff
@@ -0,0 +1,263 @@
+  densities[6] = 988.04;
+  densities[7] = 983.2;
+  densities[8] = 977.76;
+  densities[9] = 971.79;
+  densities[10] = 965.31;
+  densities[11] = 958.35; // this value is used as referense density with referense pressure 100 kPa.
+  densities[12] = 950.95;
+  densities[13] = 943.11;
+  densities[14] = 926.13;
+  densities[15] = 907.45;
+  densities[16] = 887;
+  densities[17] = 864.66;
+  densities[18] = 840.22;
+  densities[19] = 813.37;
+  densities[20] = 783.63;
+  densities[21] = 750.28;
+  densities[22] = 712.14;
+  densities[23] = 667.09;
+  densities[24] = 610.67;
+  densities[25] = 527.59;
+
+  string const tableName = functionName +  "_sat_density_table";
+  if( functionManager.hasGroup< TableFunction >( tableName ) )
+  {
+    return functionManager.getGroupPointer< TableFunction >( tableName );
+  }
+  else
+  {
+    TableFunction * const densityTable = dynamicCast< TableFunction * >( functionManager.createChild( "TableFunction", tableName ) );
+    densityTable->setTableCoordinates( temperatures );
+    densityTable->setTableValues( densities );
+    densityTable->setInterpolationMethod( TableFunction::InterpolationType::Linear );
+    return densityTable;
+  }
+}
+
+TableFunction const * makeSatPressureTable( string const & functionName,
+                                            FunctionManager & functionManager )
+{
+  array1d< array1d< real64 > > temperatures;
+  array1d< real64 > pressures;
+
+  temperatures.resize( 1 );
+  temperatures[0].resize( 26 );
+  pressures.resize( 26 );
+
+  temperatures[0][0] = 0.01;
+  temperatures[0][1] = 10;
+  temperatures[0][2] = 20;
+  temperatures[0][3] = 25;
+  temperatures[0][4] = 30;
+  temperatures[0][5] = 40;
+  temperatures[0][6] = 50;
+  temperatures[0][7] = 60;
+  temperatures[0][8] = 70;
+  temperatures[0][9] = 80;
+  temperatures[0][10] = 90;
+  temperatures[0][11] = 100;
+  temperatures[0][12] = 110;
+  temperatures[0][13] = 120;
+  temperatures[0][14] = 140;
+  temperatures[0][15] = 160;
+  temperatures[0][16] = 180;
+  temperatures[0][17] = 200;
+  temperatures[0][18] = 220;
+  temperatures[0][19] = 240;
+  temperatures[0][20] = 260;
+  temperatures[0][21] = 280;
+  temperatures[0][22] = 300;
+  temperatures[0][23] = 320;
+  temperatures[0][24] = 340;
+  temperatures[0][25] = 360;
+
+
+  pressures[0] =  612;
+  pressures[1] =  1200;
+  pressures[2] =  2300;
+  pressures[3] =  3200;
+  pressures[4] =  4200;
+  pressures[5] =  7400;
+  pressures[6] =  12400;
+  pressures[7] =  19900;
+  pressures[8] =  31200;
+  pressures[9] =  47400;
+  pressures[10] = 70200;
+  pressures[11] = 101000;
+  pressures[12] = 143000;
+  pressures[13] = 199000;
+  pressures[14] = 362000;
+  pressures[15] = 618000;
+  pressures[16] = 1000000;
+  pressures[17] = 1550000;
+  pressures[18] = 2320000;
+  pressures[19] = 3350000;
+  pressures[20] = 4690000;
+  pressures[21] = 6420000;
+  pressures[22] = 8590000;
+  pressures[23] = 11300000;
+  pressures[24] = 14600000;
+  pressures[25] = 18700000;
+
```

Comment:
The table spans quite a wide parameters space but what happens if the user inputs initial conditions that are not within the limits? Will it get an error when computing the property? Is the error easy to understand? is it Worth adding a check?

Reply:
Great question! I`ve checked - no error occurs in this case, but what happens is that the table just silently takes the closest value (instead of extrapolation, for example). This is not obvious, and should probably be reported, but somewhere in 

https://github.com/GEOSX/GEOSX/blob/72670ad80d856bb9d2f2e286bdcad0f523c2a52c/src/coreComponents/functions/TableFunction.hpp#L356

, not in the actual property code. And there will be a lot of such reports since we`ll compute properties for all grid blocks in similar conditions...

Reply:
@francoishamon We can add the option to add a point on both ends of the table. This is generally what I did when creating the apertureTables. That way the extrapolation has bounds. I don't like the idea of indefinite extrapolation..if someone gives a nonsense input, it should throw an error if it goes off the end of the table....which it doesn't do now...but it should imo.